torch/_inductor/ir.py

```diff
@@ -4577,6 +4577,10 @@ def __init__(  # type: ignore[no-untyped-def]
         self.op_overload = op_overload
         self.set_cpp_kernel_name(cpp_kernel_name)
         self.set_python_kernel_name(python_kernel_name)
+        if not ordered_kwargs_for_cpp_kernel and op_overload is not None:
+            ordered_kwargs_for_cpp_kernel = tuple(
```

Comment:
`collect_arg_kwarg_properties` already sets `ordered_kwargs_for_cpp_kernel`. The new CI failures seem relate to  that.